Fix alignment of loading message and add translation tags.

SahilSingh177 opened the pull request:
The loading message that appears at the top of the page when loading the Bots/Uploaded Files/Users/Deactivated Users page under Settings appears to be misaligned.
Fixed this by changing the top margin of the 'Loading...' message.

Also, added the translation tags for 'Loading...' message.

<!-- If the PR makes UI changes, always include one or more still screenshots to demonstrate your changes. If it seems helpful, add a screen capture of the new functionality as well.

Tooling tips: https://zulip.readthedocs.io/en/latest/tutorials/screenshot-and-gif-software.html
-->

**Screenshots and screen captures:**

<b> Alignment: </b> 
Previous behavior:

![load1](https://user-images.githubusercontent.com/96344003/218270229-e0785a59-8c13-4456-8933-fd7ac9a966b0.png)

New behavior(After code changes):

![load1](https://user-images.githubusercontent.com/96344003/224825094-6db7e28d-c9ce-4455-9ceb-fcba841d10c3.png)

![load2](https://user-images.githubusercontent.com/96344003/224825104-52d9f365-3992-4587-8de5-7c3b0ea66a89.png)

I added the translation for 'Loading...' in Hindi language to check whether translation tags are working or not and it works perfectly.
Here are the screenshots that show that translation tags been successfully implemented:

![load3](https://user-images.githubusercontent.com/96344003/224825148-9384891d-7421-42e6-b814-821e073c5e8f.png)

<details>
<summary>Self-review checklist</summary>

<!-- Prior to submitting a PR, follow our step-by-step guide to review your own code:
https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code -->

<!-- Once you create the PR, check off all the steps below that you have completed.
If any of these steps are not relevant or you have not completed, leave them unchecked.-->

- [x] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Communicate decisions, questions, and potential concerns.

- [x] Explains differences from previous plans (e.g., issue description).
- [x] Highlights technical choices and bugs encountered.
- [x] Calls out remaining decisions and concerns.
- [x] Automated tests verify logic where appropriate.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html)).

- [x] Each commit is a coherent idea.
- [x] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [x] Visual appearance of the changes.
- [x] Responsiveness and internationalization.
- [x] Strings and tooltips.
- [x] End-to-end functionality of buttons, interactions and flows.
- [x] Corner cases, error conditions, and easily imagined bugs.
</details>

sahil839 commented:
I think a better fix here would be for the indicator to have same height as the `line-height` of text which is 20px. This way we can avoid having different hard-coded margins for two elements which should be vertically aligned.

SahilSingh177 commented:
Thanks for the suggestion. I have updated the pull request with this solution. Please review and let me know if any further changes are required.

SahilSingh177 commented:
Hey @sahil839, I am following up on this pull request and would appreciate if you could review it as it has been pending for some time now.

sahil839 commented:
Sorry for the delay, got occupied with other things.

Code changes look good. I think `settings` would be a better prefix for the first commit message rather than design. Please read https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#good-summaries for details.

SahilSingh177 commented:
Thank you for your feedback! I've updated the commit message prefix to "settings" as suggested.

sahil839 commented:
Looks good to me. @timabbott ready for your review.

timabbott commented:
lgtm but we should use the special horizontal dots unicode character, e.g. `Loading…`, not 3 literal dot characters. Can you change that and post updated screenshots @SahilSingh177?

SahilSingh177 commented:
@timabbott, I have updated the pull request with the changes you suggested.

I've made changes to the PR description and added the updated screenshot below:

![load1](https://user-images.githubusercontent.com/96344003/224816779-8d3bd6fd-6662-451f-be7a-09ffbde6d80c.png)

SahilSingh177 commented:
@timabbott, I have updated the pull request with the changes you suggested.

I've made changes to the PR description and added the updated screenshot below:

![load1](https://user-images.githubusercontent.com/96344003/224816779-8d3bd6fd-6662-451f-be7a-09ffbde6d80c.png)

timabbott commented:
I'm not sure why you escaped the character as `\u2026`. I switched to putting the actual unicode character in the source like we do everywhere else and marked this to merge once CI passes again.

SahilSingh177 commented:
> I'm not sure why you escaped the character as `\u2026`. I switched to putting the actual unicode character in the source like we do everywhere else and marked this to merge once CI passes again.

I'm sorry. I misunderstood your request for a unicode character and thought you were asking for the literal representation of the character using the escape sequence "\u2026".
